django_project/stakeholder/views.py

```diff
@@ -68,3 +86,395 @@ def get_context_data(self, **kwargs):
         context['roles'] = UserRoleType.objects.all()
 
         return context
+
+
+def create_return_results(reminders):
+    """this methods constructs the object to return
+    to the front end for search tables and crud operations
+    on both notifications and reminders"""
+    search_results = []
+    for reminder in reminders:
+        search_results.append(
+            {
+                'id': reminder.id,
+                'title': reminder.title,
+                'reminder': reminder.reminder,
+                'user': str(reminder.user),
+                'organisation': str(reminder.organisation),
+                'status': reminder.status,
+                'date': datetime.strptime(
+                    str(
+                        reminder.date + timedelta(hours=2)),
```

Comment:
why do you add 2 hours here?

Reply:
I noticed when the date is saved in the db it was being saved 2 hours behind so I assumed it was timezone issue ,so when the the rows are fetched it will show the date saved in the db and i thought that would confuse the user if they saw the date 2 hours behind and they might try to adjust it thinking its an error . so now i removed the +2 hours but how can that be resolved